### PR TITLE
Update BCD to reflect that top level await is enabled starting with FF 89

### PR DESCRIPTION
**Pull request by mirunacurtean**
A checklist to help your pull request get merged faster:
- [x] Summarize your changes

Updated BCD table to reflect: 
- [top level awai](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await)t is enabled starting with FF 89. 
- Update experimental feature status to False
- update FF and FF for Android columns to show that the feature was added in FF version 89 and removed previous notes. 

- [x] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)

https://bugzilla.mozilla.org/show_bug.cgi?id=1681046

- [x] Link to related issues or pull requests, if any

Fixes https://github.com/mdn/content/issues/4313

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,903 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (ddbeck) [inline yarn.lock:1]: Delete me 🗑️

File: javascript/operators/await.json [hunk 2]
```diff
@@ -79,24 +79,10 @@
...
               "firefox": {
-                "version_added": false,
-                "notes": "Available only in nightly builds. See <a href='https://bugzil.la/1681046'>bug 1681046</a>.",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "javascript.options.experimental.top_level_await"
-                  }
-                ]
+                "version_added": "89"
```
Comment (ddbeck) [inline javascript/operators/await.json:82]: Nice cleanup! Ordinarily, I'd suggest keeping old prefs and adding new support statements, but this is a weird one with Nightly-only.

**Review by ddbeck — CHANGES_REQUESTED** [review]
Looks good, @mirunacurtean. Just need the `yarn.lock` file removed. Thank you!

File: yarn.lock (hunk 1 above)
Comment (mirunacurtean) [reply yarn.lock]: Seems like the file is autogenerated and it gets pushed along with all the other changes despite not being staged. 
I will pay attention to deleting it before making any commits from now on.

Comment (ddbeck) [reply yarn.lock]: As an alternative, you can ignore the file on your local clone. To do that, add `yarn.lock` on a new line in `.git/info/exclude` (i.e., from the root of the repository, run `echo 'yarn.lock' >> .git/info/exclude`).

**Review by ddbeck — APPROVED** [review]
Thank you! 🎉

**Pull request merged**